src/mesh/mesh_generation.C

```diff
@@ -54,6 +49,13 @@
 #include "libmesh/enum_order.h"
 #include "libmesh/int_range.h"
 #include "libmesh/parallel.h"
+#include "libmesh/parallel_ghost_sync.h"
+
+// C++ includes
+#include <cstdlib> // *must* precede <cmath> for proper std:abs() on PGI, Sun Studio CC
```

Comment:
🤡

Reply:
Yeah, I doubt that comment has remained relevant ever since someone (Ben?) started that workaround a decade ago; I can't imagine any compiler version with half-broken std::abs is managing to handle C++11 fine.  But meh, couldn't kill us to preserve non-alphabetical-order in a few places.